app/src/main/res/values-nb/strings-nb.xml

```diff
@@ -1,501 +1,848 @@
+	<string name="show_interface_hints">Vis hint for grensesnitt</string>
 	<string name="show_graph_time_lines">Vis Graf grid tidslinjer</string>
 	<string name="show_graph_glucose_lines">Vis Graf grid glukose linjer</string>
-	<string name="display_filtered_plot">Skjerm filtrert tomten</string>
-	<string name="display_raw_plot_data">Vis rådata tomten</string>
-	<string name="show_graph_target_line">Vis graf mållinje</string>
-	<string name="show_graph_recent_average">Vis Graf siste gjennomsnitt</string>
+	<string name="display_filtered_plot">Vis filtrert plott</string>
+	<string name="display_raw_plot_data">Vis rådata plott</string>
+	<string name="show_graph_target_line">Vis graf for mållinje</string>
+	<string name="show_graph_recent_average">Vis graf for Siste gjennomsnitt</string>
 	<string name="show_graph_total_average">Vis graf Total gjennomsnittlig</string>
-	<string name="show_momentum_working_curve">Vis Momentum arbeidskurven</string>
-	<string name="show_noise_workings">Vis Støy hjemkomsten</string>
-	<string name="parakeet_and_extra_features">Undulat og Ekstra testfunksjoner</string>
+	<string name="show_momentum_working_curve">Vis Momentkurven</string>
+	<string name="show_noise_workings">Vis Støyarbeid</string>
+	<string name="parakeet_and_extra_features">Parakeet og Ekstra testfunksjoner</string>
 	<string name="insulin_duration_hours">Insulin varighet i timer</string>
-	<string name="default_liver_sensitivity_ratio">Standard Liver Følsomhet Ratio</string>
-	<string name="key_is_used_instead_of_google_account">Key brukes i stedet for Google-konto</string>
-	<string name="sync_using_custom_security_key">Synkroniser bruker egendefinerte sikkerhetsnøkkel</string>
-	<string name="handset_group_security_key">Handset Konsernsikkerhet Sync Key</string>
-	<string name="handset_sync_grouping_key">Handset sync gruppering nøkkel</string>
-	<string name="alert_name_colon">Alert Navn:</string>
+	<string name="default_liver_sensitivity_ratio">Standard Lever Følsomhetsforhold</string>
+	<string name="key_is_used_instead_of_google_account">Nøkkel brukes i stedet for Google-konto</string>
+	<string name="sync_using_custom_security_key">Synkroniser med egendefinert sikkerhetsnøkkel</string>
+	<string name="handset_group_security_key">Gruppesikkerhets syknroniseringsnøkkel for håndsett</string>
+	<string name="handset_sync_grouping_key">Syknroniserings gruppenøkkel for håndsett</string>
+	<string name="alert_name_colon">Alarmnavn:</string>
 	<string name="threshold_colon">Terskel:</string>
-	<string name="default_snooze_colon">Standard Snooze:</string>
-	<string name="re_raise_every_x_minutes_if_unaknowledged">Re-raise hver\nx minutter hvis\nunaknowledged:</string>
-	<string name="alert_tone_colon">Alert Tone:</string>
+	<string name="default_snooze_colon">Standard Utsettelse:</string>
+	<string name="re_raise_every_x_minutes_if_unaknowledged">Aktiver hvert \nx-te minutter hvis \nikke b
```

Comment:
is cddata correct here??

Reply:
Line 760: 
English: <![CDATA[Notify when data a
Norwegian: <![CDATA[Gi beskjed når 

I believe this is correct. or is it another line?